src/app/core/storage/storage.service.ts

```diff
@@ -2,16 +2,17 @@ import { Injectable } from '@angular/core';
 
 @Injectable()
 export class StorageService {
+    private tokenKey: string = ENV_PUBLIC_CONFIG.localStorageTokenKey;
```

Comment:
I saw that this token name is hardcoded, and it is not passed threw build environment. It would be less frustrating to hame it in place as for me.